src/apphosting/githubConnections.ts

```diff
@@ -155,10 +170,10 @@ async function createFullyInstalledConnection(
   location: string,
   connectionId: string,
   oauthConn: devConnect.Connection,
-  withNewInstallation = false,
+  withExistingInstallationId: string | undefined = undefined,
```

Comment:
this can be changed to just `installationId` since a connection should not be created without an installationId.